Neos.Flow/Classes/ResourceManagement/EelHelper/StaticResourceHelper.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\ResourceManagement\EelHelper;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Flow\ResourceManagement\ResourceManager;
+use Neos\Flow\I18n\Service as I18nService;
+
+class StaticResourceHelper implements ProtectedContextAwareInterface
+{
+    /**
+     * @Flow\Inject
+     * @var ResourceManager
+     */
+    protected $resourceManager;
+
+    /**
+     * @Flow\Inject
+     * @var I18nService
+     */
+    protected $i18nService;
+
+    /**
+     * Get the public uri of a package resource
+     *
+     * @param string $packageKey
+     * @param string $pathAndFilename
+     * @param bool $localize
+     * @return string
```

Comment:
Since this source is used to generate the [Eel helper reference](https://neos.readthedocs.io/en/stable/References/EelHelpersReference.html) we should add descriptions for the params. Especially the `$localize` part that is a bit .. special :)

Reply:
Added some documentation